composer.json

```diff
@@ -21,44 +22,52 @@
             "role": "lead"
         }
     ],
-    "version": "4.0.0",
-    "time": "2017-11-11",
+    "time": "2021-10-08",
     "repositories": [
         {
-            "type": "pear",
-            "url": "https://pear.horde.org"
+            "type": "composer",
+            "url": "https://horde-satis.maintaina.com/"
```

Comment:
same here. I think your branch is not rebased correctly. Try to do this first

Reply:
ah thanks, ok will check

Reply:
repos diverged, I resolved the conflict in favor of upstream